From b728764896ae7d9101a4c3d3f82dffbd2be96aaa Mon Sep 17 00:00:00 2001
From: Oleksandr Shaporov <o.shaporov@inango-systems.com>
Date: Wed, 5 May 2021 14:08:29 +0300
Subject: [PATCH] Commit Title: Move all bridges to OvS

BugID                 : 23658
Bug Fixed(y/N[blankN]):
CodeReviewId          : 0
Description           : Added all Pumas7's bridges to OvS except a-mux
---
 utilities/ovs-ctl.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utilities/ovs-ctl.in b/utilities/ovs-ctl.in
index 6c86ac016..fa86f5ff1 100644
--- a/utilities/ovs-ctl.in
+++ b/utilities/ovs-ctl.in
@@ -399,7 +399,7 @@ set_defaults () {
     fi
 
     if ! test -e /nvram/openvswitch.conf; then
-        echo 'OPTIONS="--ovs-bridges=brlan0"' > /nvram/openvswitch.conf
+        echo 'OPTIONS="--ovs-bridges=*,!a-mux"' > /nvram/openvswitch.conf
     fi
 }
 
